packages/package_info_plus/package_info_plus_macos/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_info_plus_macos
 description: macOS implementation of the package_info_plus plugin.
-version: 1.3.0
+version: 2.0.0
```

Comment:
Does this need to be a breaking change?

Reply:
I wondered that too, and got this answer: https://github.com/fluttercommunity/plus_plugins/pull/1135#issuecomment-1264632986

Reply:
Ok, because of the federated plugin structure it's safer to update everything.